src/deploy-typesense.ts

```diff
@@ -50,6 +50,7 @@ const setupCollection = async function (collection_name: string, schema_filename
         name: collection_name,
         fields,
         default_sorting_field: 'sort-index',
+        token_separators: ['-', '&', '.', ',', '*', '/', '|', '+'],
```

Comment:
What about parentheses here? I feel like they might be relevant tokes in some cases.

Reply:
Do you expect parentheses in the middle of a word?

Reply:
There aren't any at the moment anyway:

![image](https://user-images.githubusercontent.com/4048809/158901404-aa3d61e3-3c77-40d1-90d4-65b1bb213ac5.png)

Reply:
So you would suggest adding them if they come up? Maybe we should write a test for that.

Reply:
No, I don't expect them to ever come up in a way that would require adding them. What case are you thinking of?

Reply:
For example weird puns in company names. Let's say a hairdresser (since they are know for their puns) calls themselves: (B)engelshaare (I know, it's bad, but I didn't have one on the top of my head and this is enough to demonstrate)
I would want that to match Engelhaare, Bengelhaare, (Bengelhaare,  B)engelhaare and (B)engelhaare

Reply:
I'm pretty sure that apart from the first one, all of these would match already. And if you would set parentheses as separators, "bengelhaare" would currently not match anymore (see https://github.com/typesense/typesense/issues/122#issuecomment-907854479), which I would consider a much more major regression.